### ISSUE TITLE
question: Is there a spec documenting the json format?

### ISSUE THREAD

**Issue by jquense** [TypeStrong/typedoc#1145]
Hi there, is there a description of what the JSON format is, or includes, Or like type defs describing it somewhere? I'd like build a thing on top of it but at the moment i'm sort of just testing it via trial and error

**Comment by Gerrit0** [comment]
Not currently, see also #1125.

PR #936 adds types for the resulting JSON, no schema exactly, but it should be helpful. I haven't had much time to work on getting it merged the past few weeks. I'm hoping I'll have a couple weeks to work on TypeDoc later this month. (I'm kind of waiting on #801 to merge #936 as there are several conflicting changes)

**Comment by Gerrit0** [comment]
Closing in favor of #1125

**Comment by jquense** [comment]
thanks for the response! I took a look at the PR and the types there, very helpful.
